package.json

```diff
@@ -7,5 +7,8 @@
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
I know that you're just pushing the POC files, but perhaps stubbing some super hello world tests would keep us honest as we're developing.  The first test is always the hardest one to get into a repo.

Reply:
Totally agree.

For setting up a test framework do you have a suggestion to which I should follow? Everyone seems to have their own style. I want to make sure it is aligned to front-end style. 

My first thought is to copy maybe the entity store repo.

Reply:
I think maybe this should be a different PR. Keep them small?

Reply:
yep, that sounds good.  I'm indifferent to what we use, but it seems like we commonly use mocha+chai YMMV